alert/alert.go

```diff
@@ -25,6 +26,7 @@ type Alert struct {
 	ServiceID string    `json:"service_id"`
 	CreatedAt time.Time `json:"created_at"`
 	Dedup     *DedupID  `json:"dedup"`
+	Meta      AlertMeta `json:"meta"`
```

Comment:
I think this should be a separate method for the store rather than bundling it all together on the struct -- we only need to make the second call if metadata is provided, and the only way to retrieve it is GraphQL which we can handle that via resolver

Reply:
I've added new method `CreateMetaDataTx` and we can pass the meta and alertId as an agrument.